README.md

```diff
@@ -37,15 +58,213 @@ To get the data from the database:
+The [instruction file](https://github.com/GeospatialResearch/Digital-Twins/blob/lidar_to_db/src/lidar/file.json) used to create hydrologically conditioned DEM is passed to the **get_dem_path(instruction)** function which checks if the DEM information exists in the database, if it doesn’t exist, geofabrics is used to generate the hydrologically conditioned DEM which gets stored in the local directory and the metadata of the generated DEM is stored in the database and file path of the generated DEM is returned which is then used to run the flood model. 
+
+## LiDAR Database
+
+The data source for the LiDAR data is [opentopography]( https://portal.opentopography.org/dataCatalog). The data for the requested catchment area is downloaded using [geoapis](https://github.com/niwa/geoapis ) in the local directory set by the user. To store the LiDAR metadata in the database, **lidar_metadata_in_db.py** script is used. The [instruction file](/lidar_to_db/src/lidar/file.json ) and path to the local directory where user wants to store the LiDAR data is passed as an argument to **store_lidar_path(file_path_to_store, instruction_file) function as shown below:**
+
+   ```
+   if __name__ == "__main__":
+    from src.digitaltwin import setup_environment
+    instruction_file = "src/lidar_test.json"
+    file_path_to_store = "your_path/lidar_data"
+    with open(instruction_file, 'r') as file_pointer:
+        instructions = json.load(file_pointer)
+    engine = setup_environment.get_database()
+    Lidar.__table__.create(bind=engine, checkfirst=True)
+    geometry_df = gpd.GeoDataFrame.from_features(instructions["features"])
+    store_lidar_path(engine, file_path_to_store, geometry_df)
+    store_tileindex(engine, file_path_to_store)
+    lidar_file = get_lidar_path(engine, geometry_df)
+    
+   ```
+
+ 
+ To get the path of the lidar file within the given catchment area:
+**store_lidar_path()** fucntion is used first in case data is not available in the database, user needs to provide database information to connect to the database, the path where Lidar data will be stored and geopandas dataframe to get the geometry information.   
+Then **store_tileindex()** function is used to store the corresponding tiles information, user needs to provide database information to connect to the database and the path where Lidar data will be stored and finally
+**get_lidar_path function()** is used which requires two arguments i.e. engine to connect to the database and geopandas dataframe to get the geometry information to get the path of the files within the catchment area. 
+
+## BoundaryConditions data
+
+### Rain gauges location
+
+The rain gauages location is accessed from [HIRDS](https://hirds.niwa.co.nz/) which is tool that provides a map-based interface to enable rainfall estimates to be provided at any location in New Zealand. The gauges information can be stored in the database using **hirds_gauges.py** script as shown below:
```

Comment:
```suggestion
`store_lidar_path()` function is used first in case data is not available in the database, user needs to provide database information to connect to the database, the path where Lidar data will be stored and geopandas dataframe to get the geometry information.   
Then `store_tileindex()` function is used to store the corresponding tiles information, user needs to provide database information to connect to the database and the path where Lidar data will be stored and finally
`get_lidar_path function()` is used which requires two arguments i.e. engine to connect to the database and geopandas dataframe to get the geometry information to get the path of the files within the catchment area. 

## BoundaryConditions data

### Rain gauges location

The rain gauges location is accessed from [HIRDS](https://hirds.niwa.co.nz/) which is tool that provides a map-based interface to enable rainfall estimates to be provided at any location in New Zealand. The gauges information can be stored in the database using `hirds_gauges.py` script as shown below:
```

Fix spelling errors and change bolded function names to monospaced

Reply:
This should be done throughout the readme